examples/rollbar-spring-boot-webmvc/src/main/java/com/example/springbootwebmvc/RollbarConfig.java

```diff
@@ -0,0 +1,43 @@
+package com.example.springbootwebmvc;
+
+import com.rollbar.notifier.Rollbar;
+import com.rollbar.notifier.config.Config;
+import com.rollbar.spring.webmvc.RollbarSpringConfigBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+
+@Configuration()
+@EnableWebMvc
+@ComponentScan({"com.example.springbootwebmvc","com.rollbar.spring.boot.webmvc"})
```

Comment:
I think that we can just use here this:

```
@ComponentScan({"com.example.springbootwebmvc","com.rollbar.spring"})
```

If not, in order to make Spring discover the `RollbarHandlerExceptionResolver` we should add to that list the `com.rollbar.spring.webmvc` that would be more verbose.

Reply:
Looks like we need the springbootwebmvc so that the ServletContextInitializer gets called in the spring-boot app

```Java
@ComponentScan({
        "com.example.springbootwebmvc",
        "com.rollbar.spring.boot.webmvc",
        "com.rollbar.spring.webmvc"
})
```

Reply:
Disregard I was able to use the simpler method:

```java
@ComponentScan({"com.example.springbootwebmvc","com.rollbar.spring"})
```